src/ContractEncoder.js

```diff
@@ -17,19 +17,15 @@ class ContractEncoder {
      * @returns {Object} Decoded contract metadata as POJO.
     */
     decode(data) {
-        const binData = this._apiEncoder.decodeWithType(data, 'contract_bytearray')
-        const decoded = RLP.decode(binData, true)
-        const stringDecoder = new TextDecoder()
+        const bytecode = this._apiEncoder.decodeWithType(data, 'contract_bytearray')
 
-        return {
-            tag: byteArray2Int(decoded.data[0]),
-            vsn: byteArray2Int(decoded.data[1]),
-            sourceHash: byteArray2Hex(decoded.data[2]),
-            aevmTypeInfo: decoded.data[3],
-            compilerVersion: stringDecoder.decode(decoded.data[5]),
-            payable: Boolean(decoded.data[6][0]),
-            bytecode: this._bytecodeSerializer.deserialize(new Uint8Array(decoded.data[4]))
-        }
+        const fateContractBytearray = Buffer.concat([
```

Comment:
I didn't able to run the browser-tests because other issue, but isn't `Buffer` a nodism?
This lib should work in a browser without transpilation/compilation

Reply:
I forgot to import "safe-buffer" which is already used as a buffer polyfill.

Reply:
I remember that I wanted to get rid of it, but didn't managed to find time to find an alternative implementation of base64, which is the only usage of it right now